debian/control

```diff
@@ -40,10 +40,13 @@ Package: snapd
 Architecture: any
 Depends: adduser,
          ca-certificates,
+         cgroup-lite,
```

Comment:
This is a universe package in xenial. So we will need to either get a MIR or only add this dependency on trusty (which will be slightly ugly using debian/control.in (but maybe nowdays there is a better way?).